docs/content/dependencies.md

```diff
@@ -142,3 +142,82 @@ would all work like [the param example in the getting started guide]({{< ref
 "getting-started.md" >}}#params). We're not quite there yet, but keep an eye
 out!
 {{< /note >}}
+
+## Locking
+
+There can be some scenarios where a group of tasks are not explicitly dependent
+on each other but also cannot be run in parallel. A good example of this is
+package management tools like
+[apk](http://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management) or
+[apt](https://wiki.debian.org/Apt). As an example, let's look at this file which
+installs three packages:
+
+```hcl
```

Comment:
in "real life" these tasks would probably all be dependent on an `apt-get update` instead of inlining it. No reason to pull thrice in a run. Let converge deduplicate :)